flatpak/desktop/pizza.isle.Isle.metainfo.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>pizza.isle.Isle</id>
+  
+  <name>Isle Decomp</name>
+  <summary>LEGO Island decompilation project</summary>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  
+  <recommends>
+    <display_length compare="ge">640</display_length>
+  </recommends>
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>gamepad</control>
+  </supports>
+  
+  <description>
+    <h1>
+        LEGO Island, portable
+    </h1>
+    <p>
+      This initiative is a portable version of LEGO Island (Version 1.1, English) based on the <a href="https://github.com/isledecomp/isle">decompilation project</a>. Our primary goal is to transform the codebase to achieve platform independence, thereby enhancing compatibility across various systems while preserving the original game&apos;s experience as faithfully as possible.
+    </p>
+    <p>
+      Please note: this project is dedicated to achieving platform independence without altering the core gameplay, adding new features, enhancing visual quality, or rewriting code for improvement&apos;s sake. While those are worthwhile objectives, they are not within the scope of this project.
+    </p>
+  </description>
+
+  <categories>
+    <category>Game</category>
+    <category>KidsGame</category>
+    <category>AdventureGame</category>
+    <category>Qt</category>
+  </categories>
+
+  <keywords>
+    <keyword translate="no">LEGO</keyword>
+    <keyword translate="no">lego</keyword>
```

Comment:
except for error messages there is no text to translate and it works with all the translated assets.